docs/docs/spacetimeauth/configure-project.md

```diff
@@ -0,0 +1,128 @@
+all the necessary information about the authenticated user.
+Here are the claims (i.e, user information available in the ID token) provided by
+each scope:
+
+| Scope             | Claims                                                                                                                                                                |
+| ----------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| openid (required) | `sub` (unique user identifier)                                                                                                                                        |
+| profile           | `name`, `family_name`, `given_name`, `middle_name`, `nickname`, `preferred_username`, `picture`, `website`, `gender`, `birthdate`, `zoneinfo`, `locale`, `updated_at` |
+| email             | `email`, `email_verified`                                                                                                                                             |
+
+You can request all or a subset of these scopes when initiating the
+authentication flow in your application.
+
+### Redirect URIs
+
+Redirect URIs are a critical part of the OAuth2 and OpenID Connect flows.
+They ensure that after a user authenticates, they are redirected back to a
+trusted location in your application.
+
+When configuring redirect URIs for your client, ensure that they match the URIs
+used in your application. This includes the scheme (http or https), domain,
+port (if applicable), and path.
+For example, if your application is hosted at `https://myapp.com` and you
+initiate the authentication flow from `https://myapp.com/login`, you might
+set the redirect URI to `https://myapp.com/callback`.
+
+To find the correct redirect URIs for your application, refer to the
+documentation of the authentication library you are using or check out our
+integration guides with various frameworks.
+
+## Setting Up Third-Party Identity Providers
+
+SpacetimeAuth supports multiple third-party identity providers, allowing users to
+authenticate using their existing accounts. Supported providers include:
+
+- Google
+- GitHub
+- Discord
+- Twitch
+- Kick
+- More providers will be added in the future
+
+User's information from third-party identity providers is mapped to the standard
+OpenID Connect claims used by SpacetimeAuth. This ensures a consistent user
+experience regardless of the identity provider used.
+For example the username claim is mapped to the standard `preferred_username` claim.
+
+You can manage identity providers by navigating to the "Identity Providers" tab
+in your SpacetimeAuth project dashboard.
+
+![Identity Providers tab](./identity-providers-tab.png)
+
+To enable an identity provider, enter the client ID and client secret obtained
+from the provider's developer console. You must also configure the redirect URI
```

Comment:
I would maybe add a note here saying that SpacetimeAuth is acting as a client for the external identity provider in this case.

Reply:
Done

> Since SpacetimeAuth acts as a client for the external identity provider,
you need to provide the client ID and client secret obtained
from the provider's developer console in order to enable the provider.
You must also configure the redirect URI in the provider's developer console to
point to SpacetimeAuth (see below).